src/background/controllers/inpageAccountController.ts

```diff
@@ -0,0 +1,42 @@
+import QryptoController from '.';
+import IController from './iController';
+import { MESSAGE_TYPE } from '../../constants';
+
+export default class InpageAccountController extends IController {
```

Comment:
I had it as that initially, thought it would be clearer to split it up since this obj might grow; I dont feel strongly about it though, do you want me to put it back into AccountController?

Reply:
your call